File: Examples/CSharp/ProgrammersGuide/WorkingWithProjects/PrintTaskWritingException.cs
using System.IO;
using Aspose.Tasks;
using System;

namespace Aspose.Tasks.Examples.CSharp.WorkingWithProjects
{
    public class PrintTaskWritingException
    {
        public static void Run()
        {
            //ExStart: PrintTaskWritingException
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_Projects();

            Project project = new Project(dataDir+ "PrintTaskWritingException.mpp");

            try
            {
                project.Save("project.MPP", Aspose.Tasks.Saving.SaveFileFormat.MPP);
            }
            catch (TasksWritingException ex)
            {
                Console.WriteLine(ex.LogText);
            }
            //ExEnd: PrintTaskWritingException
        }
    }
}

File: Examples/VisualBasic/ProgrammersGuide/WorkingWithProjects/PrintTaskWritingException.vb
Imports Microsoft.VisualBasic
Imports System.IO
Imports Aspose.Tasks
Imports System

Namespace Aspose.Tasks.Examples.VisualBasic.WorkingWithProjects
    Public Class PrintTaskWritingException
        Public Shared Sub Run()
            'ExStart: PrintTaskWritingException
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_Projects()

            Dim project As New Project(dataDir & "PrintTaskWritingException.mpp")

            Try
                project.Save("project.MPP", Aspose.Tasks.Saving.SaveFileFormat.MPP)
            Catch ex As TasksWritingException
                Console.WriteLine(ex.LogText)
            End Try
            'ExEnd: PrintTaskWritingException
        End Sub
    End Class
End Namespace